### PR TITLE
Add logic related to empty tick quorum votes

### LINKED ISSUE #23: Empty ticks validation only requires 226 votes
From @fnordspace 
```For an empty tick this is already quorum as 226 votes for an empty tick is enough. If the node is in catchup he might only receive the empty votes and move on compared to a node that was live during the tick. That one could receive a full list of votes and then due to for example auto f5 would move on.```

### PR DESCRIPTION
**Pull request by LINCKODE**
Fixes #23.
The minimum required amount of aligned votes for a an empty tick is 226 and 451 for non empty.
Adds handling for case where tick is empty and aligned votes count is >=226.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -2,6 +2,8 @@ module github.com/qubic/go-archiver-v2
 
 go 1.25.0
 
+toolchain go1.25.5
```
Comment (qubicmio) [inline go.mod]: what is this needed for?

File: validator/quorum/validator_test.go
Comment (qubicmio) [inline validator/quorum/validator_test.go:1]: Please improve the tests.

I suggest to add one test case for the validator with 226+ votes for empty tick but only 225 valid ones. That should lead to an error. And one testcase with 226 votes for empty tick that does not error.

Furthermore all the test cases from the PR require an error to be returned. That is not intuitive for tests that verify the happy path. Only the errors that we want to test should be tested. 

Additionally you check for the absence of the error messages. If the error message changes the test will never catch the problem. So the check is probably superfluous (it will not match now and it will not in the future, if the message is changed).

Could you change the testcases in a way that there are only errors in case we test a failure and also have cases that do not error?

**Review by qubicmio — CHANGES_REQUESTED** [review]
The code that handles the new case looks good but the testcases need to be improved in my opinion and a testcase for 225 valid votes out of 226 should be added.

File: validator/quorum/validator_test.go [hunk 2]
```diff
@@ -257,6 +257,80 @@ func deepCopy(votes types.QuorumVotes) types.QuorumVotes {
...
+
+	_, err := validateVotes(context.Background(), votes, computors.Computors{}, 0)
+	require.ErrorContains(t, err, "not enough quorum votes")
+}
+
+func TestValidateVotes_EmptyTick_BelowAlignmentThreshold(t *testing.T) {
+	// 226 empty-tick votes total but one is misaligned, so only 225 align — one below
+	// the relaxed empty-tick threshold. Must be rejected at the post-alignment count gate.
+	votes := buildAlignedVotes(EmptyTickMinVoteCount, [32]byte{})
+	votes[0].Second += 1 // breaks alignment for this vote only
```
Comment (qubicmio) [inline validator/quorum/validator_test.go:305]: what I meant was 226 aligned votes but one without valid signature. So that we make sure this check still works for empty ticks. (basically the check `err = quorumTickSigVerify(ctx, alignedVotes, computors, targetTickVoteSignature)` should fail after getting the aligend votes, if I understood the code correctly.

But keep this test case. It's useful, too.

**Review by qubicmio — CHANGES_REQUESTED** [review]

File: validator/quorum/validator_test.go (hunk 2 above)
Comment (LINCKODE) [reply validator/quorum/validator_test.go:305]: Testing signatures cryptographically is not feasible as discussed internally.
